### DISCUSSION TITLE
Content Editor Branching

### DISCUSSION THREAD

**Discussion by Perni1984** [tinacms/tinacms#2176] [Help] [answered]
@jbevis @dwalkr @jeffsee55 we are currently in the last steps of finishing our first tina powered website. As we have a team of content editors that would like to have previewable urls a branching solution would be the last missing key part for our deployment. Is there anything we can do to help you guys on the branching feature (e.g. https://github.com/tinacms/tinacms/pull/2129)?

Our current idea of a workaround would be to have dedicated subdomain urls for each editor and using dedicated branches for each editor, where we pass in the branch via an environment variable in the vercel environment - but if you have a better idea - we are very open to it.

**Comment by jeffsee55** [comment, marked as answer]
We're currently working on branching and hope to deliver it under a feature flag this quarter. Having said that, it'll likely be a pretty low-level tool that will still require manually merging those branches back into your "production" branch. We should have that in a viewable state in the next couple of weeks so we'll ping you to give it a try

**Reply by Perni1984** [reply]
We didn't expect to have a full fledged version with bells and whistles, rather than having a way so our content managers don't need to create branches themselves. Sugar on top would be if there is a draft PR/PR automatically created in Github when a new branch is created, but I am sure we could automize that with a bot or github action on our side aswell.

**Reply by Perni1984** [reply]
I guess this (https://github.com/marketplace/actions/github-action-for-creating-pull-requests) would do the job.
